package.json

```
@@ -52,6 +71,18 @@
     },
     "rules": {}
   },
+  "jest": {
+    "transform": {
+      "^.+\\.js$": "babel-jest",
+      "^.+\\.vue$": "vue-jest"
+    },
+    "modulePathIgnorePatterns": [
+      "<rootDir>/dist"
+    ],
+    "moduleNameMapper": {
+      ".*vue-slider-component/theme/default.css": "<rootDir>/src/config/emptyStubForJSLinter.js"
```

```suggestion
      "\.css$": "<rootDir>/src/config/emptyStubForJSLinter.js"
```
There's no reason to import any other CSS, is there?

---

I hardwired the filename thinking that I would generalize when we got there, but fine.

The pattern needs an extra backslash.